src/js/components/stock-movement/modals/RejectRequestModalHeader.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import Translate from 'utils/Translate';
+
+const RejectRequestModalHeader = ({ id }) => (
+  <div className="header">
+    <h4 className="title">
+      <Translate
+        id="react.autosaveFeatureModal.title.label"
+        defaultMessage={`Please provide a reason for rejecting request: ${id}`}
```

Comment:
At first I wanted to write that from the user perspective in my opinion it should display a code of a request, not id, but then I realized you do it, so maybe not to cause confusions either rename the prop to `identifier` (as it is returned from the backend) or even `requestCode`, but I think we should follow the first approach - so to keep the names the same backend <=> frontend - in this case -> `identifier`